src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/RazorDocumentExcerptService.cs

```diff
@@ -53,83 +47,46 @@ public RazorDocumentExcerptService(DocumentSnapshot document, IRazorSpanMappingS
                 return null;
             }
 
-            var mapped = await _mapper.MapSpansAsync(document, new[] { span }, cancellationToken).ConfigureAwait(false);
+            var mapped = await _mappingService.MapSpansAsync(document, new[] { span }, cancellationToken).ConfigureAwait(false);
```

Comment:
Seems like we'll only every pass one span? If so we shouldn't give it an array.
```suggestion
            var mapped = await _mappingService.MapSpansAsync(document, span, cancellationToken).ConfigureAwait(false);
```

Reply:
https://github.com/dotnet/aspnetcore-tooling/pull/2041#discussion_r441888540